client/state/selectors/index.js

```diff
@@ -162,14 +163,14 @@ export isFollowing from './is-following';
 export isHiddenSite from './is-hidden-site';
 export isFetchingPublicizeShareActionsScheduled from './is-fetching-publicize-share-actions-scheduled';
 export isFetchingPublicizeShareActionsPublished from './is-fetching-publicize-share-actions-published';
-export isSchedulingPublicizeShareAction from './is-scheduling-publicize-share-action';
 export isJetpackModuleActive from './is-jetpack-module-active';
 export isJetpackModuleUnavailableInDevelopmentMode from './is-jetpack-module-unavailable-in-development-mode';
 export isJetpackSettingsSaveFailure from './is-jetpack-settings-save-failure';
 export isJetpackSiteConnected from './is-jetpack-site-connected';
 export isJetpackSiteInDevelopmentMode from './is-jetpack-site-in-development-mode';
 export isJetpackSiteInStagingMode from './is-jetpack-site-in-staging-mode';
 export isJetpackUserMaster from './is-jetpack-user-master';
+export isMainSiteToSecondarySiteConnection from './is-main-site-to-secondary-site-connection';
```

Comment:
I really think we should find a better name for that one. Maybe @gwwar might have good ideas.